src/domain.module.ts

```diff
@@ -21,6 +25,10 @@ import { JsonSchemaService } from './common/schemas/json-schema.service';
     { provide: IBalancesRepository, useClass: BalancesRepository },
     { provide: IChainsRepository, useClass: ChainsRepository },
     { provide: IExchangeRepository, useClass: ExchangeRepository },
+    BackboneValidator,
+    BalancesValidator,
+    ExchangeValidator,
+    ChainsValidator,
```

Comment:
Maybe we need to consider a module for each "sub-domain" I can expect that the list is only going to grow 😅

Reply:
I totally agree, will address this in a separate PR if you are ok with it 🙂